ui/components/data-sources/AddDataSourceDialog.js

```diff
@@ -9,7 +9,16 @@ const INITIAL_STATE = {
     name: "",
     type: DataSourceType.InMemory,
     options: {
-        timestampData: true
+        // InMemory Options
+        timestampData: true,
+
+        // Postgres Options
+        url: "",
+        port: 5432,
```

Comment:
Could it ever happen that this would be configured to be something different by the user?